cli/azd/pkg/infra/provisioning/bicep/bicep_provider.go

```diff
@@ -585,6 +599,30 @@ func (p *BicepProvider) getAppConfigsToPurge(
 	return configs, nil
 }
 
+func (p *BicepProvider) getAPIManagementsToPurge(
+	ctx context.Context,
+	groupedResources map[string][]azcli.AzCliResource,
+) ([]*azcli.AzCliAPIM, error) {
```

Comment:
```suggestion
) ([]*azcli.AzCliApim, error) {
```
It is my understanding that we don't use all-caps for such cases.